cmd/collector/app/metrics.go

```diff
@@ -58,9 +58,13 @@ type SpanProcessorMetrics struct {
 	InQueueLatency metrics.Timer
 	// SpansDropped measures the number of spans we discarded because the queue was full
 	SpansDropped metrics.Counter
+	// SpansBytes records how many bytes were processed
+	SpansBytes metrics.Gauge
```

Comment:
From the comment sounds like this should be a counter.

Reply:
Sounds like it, but I'm keeping track of this in a local var, and this metric reflects the value of that var.